packages/azure-openapi-validator/autorest/package.json

```diff
@@ -56,7 +56,7 @@
     "@stoplight/spectral-ruleset-bundler": "^1.2.1",
     "@stoplight/spectral-rulesets": "~1.6.0",
     "@stoplight/spectral-runtime": "^1.1.0",
-    "@stoplight/json-ref-resolver": "3.1.3",
+    "@stoplight/json-ref-resolver": "^3.1.6",
```

Comment:
- Failures when `3.1.3` and `3.1.6` were loaded side-by-side
- Unclear from git history why this dep was pinned when related deps floated